storage/rocksdb/ha_rocksdb.cc

```diff
@@ -492,6 +492,7 @@ static my_bool rocksdb_large_prefix = 0;
 static my_bool rocksdb_allow_to_start_after_corruption = 0;
 static uint32_t rocksdb_write_policy =
     rocksdb::TxnDBWritePolicy::WRITE_COMMITTED;
+static my_bool rocksdb_error_on_suboptimal_collation = 1;
```

Comment:
s/1/TRUE

Reply:
Not in upstream fb, 0/1 is the standard. We 'fix' it on merge downstream.